controllers/useraccount/console_usersettings_test.go

```diff
@@ -0,0 +1,285 @@
+package useraccount
+
+import (
+	"context"
+	"fmt"
+	"github.com/codeready-toolchain/toolchain-common/pkg/test"
+	"github.com/stretchr/testify/require"
+	corev1 "k8s.io/api/core/v1"
+	rbac "k8s.io/api/rbac/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/client/fake"
+	"testing"
+)
+
+func TestDeleteConsoleSettingObjects(t *testing.T) {
+	t.Run("Delete ConfigMap", func(t *testing.T) {
+		t.Run("Configmap found by name and deleted", func(t *testing.T) {
+			ctx := context.Background()
+			cm := &corev1.ConfigMap{
+				TypeMeta: metav1.TypeMeta{Kind: "ConfigMap"},
+				ObjectMeta: metav1.ObjectMeta{
+					Name:      "user-settings-johnsmith",
+					Namespace: UserSettingNS,
+				},
+			}
+			cl := fake.NewClientBuilder().WithObjects(cm).Build()
+
+			err := deleteResource(ctx, cl, "johnsmith", cm)
+			require.NoError(t, err)
+			// check that the configmap doesn't exist anymore
+			err = cl.Get(ctx, client.ObjectKey{Name: "user-settings-johnsmith", Namespace: UserSettingNS}, &corev1.ConfigMap{})
+			require.True(t, errors.IsNotFound(err))
+		})
```

Comment:
Could you please add the missing `// given` `// when` `// then` comments to all tests? It's much easier to read them
```suggestion
		t.Run("Configmap found by name and deleted", func(t *testing.T) {
		        // given
			ctx := context.Background()
			cm := &corev1.ConfigMap{
				TypeMeta: metav1.TypeMeta{Kind: "ConfigMap"},
				ObjectMeta: metav1.ObjectMeta{
					Name:      "user-settings-johnsmith",
					Namespace: UserSettingNS,
				},
			}
			cl := fake.NewClientBuilder().WithObjects(cm).Build()
			
			// when
			err := deleteResource(ctx, cl, "johnsmith", cm)
			
			// then
			require.NoError(t, err)
			// check that the configmap doesn't exist anymore
			err = cl.Get(ctx, client.ObjectKey{Name: "user-settings-johnsmith", Namespace: UserSettingNS}, &corev1.ConfigMap{})
			require.True(t, errors.IsNotFound(err))
		})
```

Reply:
done in [9fb025c](https://github.com/codeready-toolchain/member-operator/pull/568/commits/9fb025c2975f0a6a69b8c4936d252bb1c47be5cf)